### PR TITLE
ECS+linkerd+Consul example setup

### PR DESCRIPTION
**Pull request by siggy**
This demonstrates setting up an ECS Cluster with linkerd for routing and
Consul for service discovery.

### CODE REVIEW

File: aws/README.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+for service discovery.
+
+The instructions below assume you have already configured AWS with the proper
+IAM, key pairs, and VPCs for an ECS cluster. For more information start here:
+http://docs.aws.amazon.com/AmazonECS/latest/developerguide/get-set-up-for-amazon-ecs.html
+
+## Overview
+
+The following components make up the system:
+* `ECS`: Docker container management. Every ECS instance runs the following
```
Comment (adleong) [inline aws/README.md]: I think we should give a little bit more background here explaining that ECS doesn't have a 1-per-node notion and so to get 1-per-node we have to run these outside of the ECS scheduler and launch them directly through the launch configuration.

File: aws/README.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+  * `linkerd`: proxies requests to hello-world
+  * `consul-agent`: local service discovery agent
+  * [`consul-registrator`](https://github.com/gliderlabs/registrator): bridge
+  between Docker and Consul, automatically registers services with consul
+* `hello-world`: example ECS task deployed separately from foundational
+  ECS+linkerd+Consul configuration, composed of a 'hello' service and a 'world'
+  service
+* `consul-server`: service discovery back-end, runs on a single EC2 instance
+
+```
```
Comment (adleong) [inline aws/README.md]: Paging @sterlingwhite

**Comment by siggy** [comment]
@dario-simonetti @kevholditch i'm working on a blog post based on this example. would love your feedback on it prior to publishing. if interested, ping me at siggy at buoyant.io and i'll invite you to the google doc.

**Comment by dario-simonetti** [comment]
Hey @siggy, sure, just done so! Happy to help

**Pull request merged**